wolfcrypt/src/ecc.c

```diff
@@ -5908,6 +5925,11 @@ int wc_ecc_init_id(ecc_key* key, unsigned char* id, int len, void* heap,
     if (ret == 0 && (len < 0 || len > ECC_MAX_ID_LEN))
         ret = BUFFER_E;
 
+#if defined(HAVE_PKCS11)
+    XMEMSET(key, 0, sizeof(ecc_key));
```

Comment:
This will need changed `wc_ecc_init_ex` does a memset. Set the isPkcs11 after the init call.

Reply:
Kept it the same for reasons discussed on slack